test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua

```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL to omit "autoCompleteText" param at request to HMI
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param and without (omited) <autoCompleteText> param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
```

Comment:
@Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..." 
2. please update description with particular resultCode from HMI (please specify "SUCCESS") and not <resultCode_received_from _HMI>